connectors/formats.mdx

```diff
@@ -0,0 +1,115 @@
+
+| Option | Description | Default |
+| ------ | ----------- | ------- |
+| `format` | The format of the data. May be one of `json` or `debezium_json` | `json` |
+| `json.confluent_schema_registry` | Set to true if data was produced by (or will be consumed via) a [Confluent Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html) connected source | `false` |
+| `json.include_schema` | Set to true to include the schema in the output, allowing it to be used with Kafka Connect connectors that require a schema | `false` |
+| `json.unstructured` | Set to true to treat the data as unstructured JSON, which will be parsed as a single column of type `TEXT` | `false` |
+| `json.timestamp_format` | The format of timestamps in the data. May be one of `rfc3339` or `unix_millis` | `rfc3339` |
+
+Json data can be either structured or unstructured. Structured data is parsed
+into columns according to the schema. Schemas can be specified via the fields in
+a SQL CREATE TABLE statement (as described in the [DDL docs](/sql/ddl)) or
+imported via a [json-schema](https://json-schema.org/) definition.
+
+Note that json-schema is a very flexible format, and not all of its features can
+be cleanly mapped to a SQL table. As a fallback, any fields that cannot be
+directly supported by Arroyo will be rendered as a single column of type `TEXT`
+with JSON-serialized data in it.
+
+Unstructured data is treated as a single column named `value` with type `TEXT`, which can be operated on using
+[SQL json functions](/sql/scalar-functions#json-functions) or UDFs.
+
+## Avro
+
+[Avro](https://avro.apache.org/) is a binary data format that is commonly used for
+data applications. Avro is a schema-based format, and requires readers and writers to
+have access to the schema in order to read and write data.
+
+Avro is well-supported by the Kafka ecosystem and the
+[Confluent Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html),
+is a popular choice for storing and serving Avro schemas. Arroyo has deep support for the Schema Registry.
+
+The following options are supported:
+
+| Option | Description | Default |
+| ------ | ----------- | ------- |
+| `format` | The format of the data. Must be `avro` | `avro` |
+| `avro.confluent_schema_registry` | Set to true if data was produced by (or will be consumed via) a [Confluent Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html) connected source | `false` |
+| `avro.raw_datums` | Set to true to serialize and deserialize as raw Avro datums instead of complete Avro records | `false` |
+| `avro.into_unstructured_json` | Convert the avro record to JSON | `false` |
+
+Avro data can be serialized/deserilaized in two ways:
+
+* As a complete Avro document, which includes the schema and metadata
+* As a raw Avro datum, which is just the data itself
+
+In the former mode, applications reading from the data will need to have access to the exact schema
```

Comment:
The "former mode"/"latter mode" don't seem to match the order you list the serialization options.